src/components/shared/input/index.tsx

```diff
@@ -86,3 +103,15 @@ const Input = <
 };
 
 export default Input;
+
+Input.defaultProps = {
```

Comment:
Remove this default props and add it to the props declaration

Reply:
Done.